frontend/viewer/src/lib/hooks/is-mobile.svelte.ts

```
@@ -1,9 +1,14 @@
 
-const MOBILE_BREAKPOINT = 768;
-
 export class IsMobile extends MediaQuery {
   constructor() {
     super(`max-width: ${MOBILE_BREAKPOINT - 1}px`);
   }
+
+  private static isMobile = new IsMobile();
+
+  static get value(): boolean {
+    return this.isMobile.current;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)